src/pages/ReimbursementAccount/ConnectBankAccount/ConnectBankAccount.js

```diff
@@ -0,0 +1,104 @@
+import lodashGet from 'lodash/get';
+import PropTypes from 'prop-types';
+import React from 'react';
+import {View} from 'react-native';
+import {withOnyx} from 'react-native-onyx';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import ScreenWrapper from '@components/ScreenWrapper';
+import Text from '@components/Text';
+import TextLink from '@components/TextLink';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import BankAccount from '@libs/models/BankAccount';
+import EnableBankAccount from '@pages/ReimbursementAccount/EnableBankAccount/EnableBankAccount';
+import * as ReimbursementAccountProps from '@pages/ReimbursementAccount/reimbursementAccountPropTypes';
+import * as Report from '@userActions/Report';
+import ONYXKEYS from '@src/ONYXKEYS';
+import BankAccountValidationForm from './components/BankAccountValidationForm';
+import FinishChatCard from './components/FinishChatCard';
+
+const propTypes = {
+    /** Bank account currently in setup */
+    reimbursementAccount: ReimbursementAccountProps.reimbursementAccountPropTypes.isRequired,
+
+    onBackButtonPress: PropTypes.func.isRequired,
+
+    /** User's account who is setting up bank account */
+    account: PropTypes.shape({
+        /** If user has two-factor authentication enabled */
+        requiresTwoFactorAuth: PropTypes.bool,
+    }),
+};
+
+const defaultProps = {
+    account: {
+        requiresTwoFactorAuth: false,
+    },
+};
+
+function ConnectBankAccount({reimbursementAccount, onBackButtonPress, account}) {
+    const styles = useThemeStyles();
+    const {translate} = useLocalize();
+
+    const bankAccountState = lodashGet(reimbursementAccount, 'achData.state');
```

Comment:
no default values for all the gets?

Reply:
it's copied from previous implementation, so not sure what default value supposed to be.

Reply:
surely any value in the matching type will be better than undefined?

Reply:
the question is what should be default 😄

Reply:
something that overlaps with the type of the value we're getting. So empty string, empty object or w/e